### PR TITLE
feat(cli/help): added codecs support section

### PR DESCRIPTION
**Pull request by SalOne22**
@Mikachu2333 should I add more information?

### CODE REVIEW

File: src/cli.rs
Comment (Mikachu2333) [inline src/cli.rs:1]: I would like to know if Rust supports displaying the syntax in Markdown format directly, or we should re-format it to make the display of the command-line interface clear and easy to understand

File: src/cli.rs [hunk 1]
```diff
@@ -9,7 +10,29 @@ pub mod preprocessors;
 pub mod utils;
 
 pub fn cli() -> Command {
-    command!().arg_required_else_help(true).codecs()
+    command!()
+        .arg_required_else_help(true)
+        .after_help(indoc! {r#"List of supported codecs
```
Comment (Mikachu2333) [inline src/cli.rs:15]: I would like to know if Rust supports displaying the syntax in Markdown format directly, or we should re-format it to make the display of the command-line interface clear and easy to understand.

Comment (SalOne22) [reply src/cli.rs:15]: We should format it by hand

**Comment by Mikachu2333** [comment]
close #191

**Comment by SalOne22** [comment]
> I'd like to help but I don't have the permission to modify and update the feature branch...

Sent invitation to collaborators

**Comment by Mikachu2333** [comment]
Need re-check

File: src/cli.rs [hunk 2]
```diff
@@ -14,19 +14,20 @@ pub fn cli() -> Command {
...
-        | png          | zune-png      | oxipng or zune-png      |
-        | avif         | libavif       | ravif                   |
-        | webp         | webp          | webp                    |
-        | ppm          | zune-ppm      | zune-ppm                |
-        | qoi          | zune-qoi      | zune-qoi                |
-        | farbfeld     | zune-farbfeld | zune-farbfeld           |
-        | psd          | zune-psd      | X                       |
-        | jpeg-xl      | jxl-oxide     | zune-jpegxl             |
-        | hdr          | zune-hdr      | zune-hdr                |
+| Image Format  | Input | Output | Note        |
```
Comment (SalOne22) [inline src/cli.rs]: encoder/decoder variant I like more, because one format can have multiple encoders

File: src/cli.rs [hunk 3]
```diff
@@ -14,25 +14,29 @@ pub fn cli() -> Command {
...
-
-        - Resize
-        - Quantization"#})
+
+| Image Format  | Input | Output | Note            |
+| ------------- | ----- | ------ | --------------- |
+| avif          | O     | O      | Static only     |
+| bmp           | O     | X      |                 |
+| farbfeld      | O     | O      |                 |
+| gif           | O     | X      | Static only     |
```
Comment (SalOne22) [inline src/cli.rs:23]: Also moving from image crate to zune-image introduced regressions in input formats, gif, ico and tiff are not supported. I can add these as a separate task

File: src/cli.rs [hunk 4]
```diff
@@ -9,7 +10,31 @@ pub mod preprocessors;
...
+
+| Image Format  | Input | Output | Note            |
+| ------------- | ----- | ------ | --------------- |
+| avif          | O     | O      | Static only     |
+| bmp           | O     | X      |                 |
+| farbfeld      | O     | O      |                 |
+| hdr           | O     | O      |                 |
+| jpeg          | O     | O      |                 |
+| jpeg-xl       | O     | O      |                 |
+| mozjpeg(jpeg) | O     | O      |                 |
```
Comment (Mikachu2333) [inline src/cli.rs]: Need to consistent with actual commands

**Review by Mikachu2333 — COMMENTED** [review]
Everything done.

**Review by Mikachu2333 — COMMENTED** [review]
If there're no other feature updates, I think it's time to merge

**Pull request merged**